src/seedu/addressbook/data/person/Address/Block.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.data.person.Address;
+
+public class Block {
```

Comment:
This class and its non-trivial methods should have header comments. 
If not, developers who write client code for this class will not know how to use it.